gradle.properties

```diff
@@ -1,3 +1,4 @@
 ideaVersion = IC-15.0.2
 javaVersion = 1.6
 ijPluginRepoChannel = alpha
+version = 0.9.1-alpha-SNAPSHOT
```

Comment:
Any obvious way to reference this from our Java code? e.g. for User-Agent string?